Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -502,6 +504,14 @@ def process_mirror_or_unknown_message(message: str) -> dict:
     return create_adaptive_card(body)
 
 
+def is_teams_ask_message(msg: str) -> bool:
+    try:
+        message: dict = json.loads(msg)
+        return set(message.keys()) == MS_TEAMS_ASK_MESSAGE_KEYS 
```

Comment:
```suggestion
        return message.keys() == MS_TEAMS_ASK_MESSAGE_KEYS 
```
my bad 😅 it's actually comparable as a set